aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -895,6 +914,11 @@ module aptos_framework::stake {
         let len = vector::length(&validator_set.active_validators);
         while (i < len) {
             let validator = vector::borrow(&validator_set.active_validators, i);
+            spec {
+                assume exists<StakePool>(validator.addr);
+                assume exists<ValidatorConfig>(validator.addr);
+                assume global<ValidatorConfig>(validator.addr).validator_index < len(validator_perf.validators);
```

Comment:
Why is this an assumption and not invariant?

Reply:
This assumption should turn into an invariant eventually. Currently, Prover cannot prove this due to the lack of loop invariants and some Prover's internal issue. These should be resolved soon.

Reply:
How soon? Are there already outstanding PRs to add support for loop variants and/or fix the internal issues in Prover?

Reply:
We found a workaround for one issue (https://github.com/move-language/move/issues/464), and are working on a fix for another issue (https://github.com/move-language/move/issues/466).